source/dlangbot/github.d

```diff
@@ -57,11 +57,26 @@ Please see [CONTRIBUTING.md](https://github.com/%s/blob/master/CONTRIBUTING.md)
 
 `, pr.user.login, pr.repoSlug);
 
-    if (refs.length)
+    static immutable bugzillaReferences = ["dlang/dmd", "dlang/druntime", "dlang/phobos",
+                     "dlang/dlang.org", "dlang/tools", "dlang/installer"];
```

Comment:
The Bugzilla integration isn't set up for these repos and they aren't looked at by the changelog generator.